### ISSUE TITLE
custom-checkboxes not visible whith high contrast on windows 

### ISSUE THREAD

**Issue by liyokuna** [twbs/bootstrap#26892]
Hello,

We discovered an issue about the custom checkbox and radio button when using a different contrast.
On basic mode, we can see [this](https://getbootstrap.com/docs/4.1/components/forms/#checkboxes), image below. ![forms-checkboxes-boostrap](https://user-images.githubusercontent.com/11217936/42750721-83f4fdec-88e8-11e8-8a12-4b842c643b45.PNG)

When changing the contrast and pick in my case the second contrast on windows, as shown on the image below.
![contraste-picked-2](https://user-images.githubusercontent.com/11217936/42750809-df16ae3c-88e8-11e8-8ca9-ae4ecf59c264.PNG)

The result of this change affects a lot the custom checkbox and when can't see anything at all, as you can see on the image below. 
![forms-checkboxes-boostrap-contrast](https://user-images.githubusercontent.com/11217936/42750843-01917690-88e9-11e8-9ac8-e2a5a23ef7f3.PNG)

Just by following those steps, you can reproduce that issue.

My compagny uses a fork of Bootstrap, named [Boosted](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap), and a colleague raises this issue, [here's the link](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/87).

Plenty of idea crossed my mind, the first one is to create a class checked and it will be manageable by JS. In this  JS file, we can checked the contrast by using Modernizr. [As this example shows the use of Modernizr with contrast](https://www.ffoodd.fr/un-test-modernizr-pour-les-contrastes-eleves/)
The second is to edit the css class in the custom-forms.scss.

If you have another idea to solve it, share it :) !

- Operating system and version (Windows 7)
- Browser and version (Firefox 52.9.0 , IE 11)

**Comment by liyokuna** [comment]
Hello,
I solved by editing the CSS classes used for checkboxes and radio button. Classes using attributes :before and :after.
I set the size border for the border and the tick for those two different elements. 
the result is this :

![](https://user-images.githubusercontent.com/11217936/42954408-ea5cd53c-8b7c-11e8-8642-efffd3ea4989.PNG)

**Comment by mdo** [comment]
Confirming these are fixed, I see borders and background images in our docs in high contrast. This was addressed at some point when we added borders to the form elements. Closing out.
